bootstrap/server.go

```diff
@@ -83,8 +83,10 @@ type bootstrapServer struct {
 var _ common.StarterShutdowner = (*bootstrapServer)(nil)
 
 func newBootstrapServer(l *logrus.Logger, b *Bootstrapd, db *sql.DB, conf *ConfigFile) (*bootstrapServer, error) {
-	grpcServer := common.NewDBGRPCServer(db)
+	grpcServer := common.NewDBGRPCServer(conf.Insecure, db)
 	ccmsg.RegisterNodeBootstrapdServer(grpcServer, &grpcBootstrapServer{bootstrap: b})
+	// Enable to observe gRPC latency
+	// grpc_prometheus.EnableHandlingTimeHistogram()
```

Comment:
Why are we leaving this in?

Reply:
To have fast access to that until having a configuration parameter for this in LogPipe.